core/src/test/java/com/linecorp/armeria/internal/common/SchemeAndAuthorityTest.java

```diff
@@ -25,13 +25,15 @@
 class SchemeAndAuthorityTest {
     @ParameterizedTest
     @CsvSource({
-            "0.0.0.0:80,        0.0.0.0:80,         0.0.0.0,            80",    // IPv4
-            "[::1]:8080,        [::1]:8080,         [::1],              8080",  // IPv6
-            "unix%3Afoo.sock,   unix%3Afoo.sock,    unix%3Afoo.sock,    -1",    // Domain socket
-            "foo.bar,           foo.bar,            foo.bar,            -1",    // Only host
-            "foo:,              foo:,               foo,                -1",    // Empty port
-            "bar:80,            bar:80,             bar,                80",    // Host and port
-            "foo@bar:80,        bar:80,             bar,                80",    // Userinfo and host and port
+            "0.0.0.0:80,        0.0.0.0:80,          0.0.0.0,            80",    // IPv4
+            "[::1]:8080,        [::1]:8080,          [::1],              8080",  // IPv6
```

Comment:
I realize that we don't have the cases for the IP addresses without port:

```suggestion
            "0.0.0.0,           0.0.0.0,             0.0.0.0,            -1",    // IPv4
            "0.0.0.0:,          0.0.0.0:,            0.0.0.0,            -1",    // IPv4 with empty port
            "0.0.0.0:80,        0.0.0.0:80,          0.0.0.0,            80",    // IPv4 with port 
            "[::1],             [::1],               [::1],              -1",    // IPv6
            "[::1]:,            [::1]:,              [::1],              -1",    // IPv6 with empty port
            "[::1]:8080,        [::1]:8080,          [::1],              8080",  // IPv6 with port
```